scripts/triggerShortcut.ts

```diff
@@ -3,19 +3,47 @@
 import yargs from 'yargs'
 import { Address, createWalletClient, http, createPublicClient } from 'viem'
 import { mnemonicToAccount } from 'viem/accounts'
-import { celo } from 'viem/chains'
+import {
+  arbitrum,
+  arbitrumSepolia,
+  base,
+  baseSepolia,
+  celo,
+  celoAlfajores,
+  mainnet,
+  optimism,
+  optimismSepolia,
+  polygon,
+  polygonAmoy,
+  sepolia,
+} from 'viem/chains'
 import { getShortcuts } from '../src/runtime/getShortcuts'
 import { NetworkId } from '../src/types/networkId'
 import { z } from 'zod'
 
 const CELO_DERIVATION_PATH = "m/44'/52752'/0'/0/0"
 
+const NETWORK_ID_TO_CHAIN = {
```

Comment:
This way TS will catch when we add a new network and forget to add it here:
```suggestion
const NETWORK_ID_TO_CHAIN: Record<NetworkId, Chain> = {
```